deploy/helm/elastic-agent/templates/integrations/_system/_system_metrics.tpl

```diff
@@ -0,0 +1,103 @@
+{{- define "elasticagent.system.config.metrics.init" -}}
+{{- if $.Values.system.metrics.enabled}}
+{{- $preset := $.Values.agent.presets.perNode -}}
+{{- $inputVal := (include "elasticagent.system.config.metrics.input" $ | fromYamlArray) -}}
+{{- include "elasticagent.preset.mutate.inputs" (list $ $preset $inputVal) -}}
+{{- include "elasticagent.preset.applyOnce" (list $ $preset "elasticagent.kubernetes.pernode.preset") -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "elasticagent.system.config.metrics.input" -}}
+- id: system-metrics
+  type: system/metrics
+  use_output: {{ $.Values.system.output }}
+  data_stream:
+    namespace: {{ $.Values.system.namespace }}
+  streams:
+    - data_stream:
+        dataset: system.cpu
+        type: metrics
+      period: 10s
+      cpu.metrics:
+        - percentages
+        - normalized_percentages
+      metricsets:
+        - cpu
+    - data_stream:
+        dataset: system.diskio
+        type: metrics
+      period: 10s
+      diskio.include_devices: null
```

Comment:
Are these nulls intentional? I would expect this to either be an empty list or for the key to not exist at all.

Reply:
this crossed my mind as well, but then I wondered why this config was rendered like that from Kibana; thus, I figured not to mess with it 😄 but sure this can be further optimised in the future